lua/weapons/cw_base/cl_model.lua

```diff
@@ -78,8 +78,13 @@ function SWEP:GetTracerOrigin()
 	if self.dt.State == CW_AIMING and self.SimulateCenterMuzzle then
 		return self.CenterPos
 	end
+
+	local attachmentTbl = self:getMuzzlePosition()
+	if not attachmentTbl or not attachmentTbl.Pos then

```

Comment:
Not a fan of double `not` checks like this.
I'd prefer:
- Two separate early returns
- Parenthesis around each side